libpod/container.go

```diff
@@ -534,6 +538,89 @@ func (c *Container) refresh() error {
 	return nil
 }
 
+// GetCommand returns command to be executes inside of container
+// ToDO need to add Command handling to libpod
+func (c *Container) GetCommand() string {
+	return ""
+}
+
+// GetAnnotations returns annotations associated with container
+// ToDO need to add Annotation handling to libpod
+func (c *Container) GetAnnotations() []string {
```

Comment:
We already have this. They're in the OCI spec, which you can get with container.Spec()

Reply:
Fixed